src/components/ui4/ButtonUi4.tsx

```diff
@@ -106,28 +106,32 @@ export function ButtonUi4(props: Props) {
       </EdgeText>
     )
 
+  // Use margin props as padding for the invisible container to increase
+  // tappable area while visually looking like margins
+  const customMarginPadding = React.useMemo(() => {
+    if (marginRem == null) return undefined
+
+    const { marginLeft, marginRight, marginTop, marginBottom } = sidesToMargin(mapSides(fixSides(marginRem, 0), theme.rem))
+    return {
+      // Use margin as padding to increase tappable area
+      paddingLeft: marginLeft,
+      paddingRight: marginRight,
+      paddingTop: marginTop,
+      paddingBottom: marginBottom
+    }
```

Comment:
Why rename all the fields? Just use `sidesToPadding` to begin with. That's what the utility is for.